src/components/Picker/index.js

```diff
@@ -58,13 +78,14 @@ class Picker extends PureComponent {
                         onClose={() => this.setState({isOpen: false})}
                         disabled={this.props.isDisabled}
                         focused={this.state.isOpen}
-                        hasError={this.props.hasError}
+                        errorText={this.props.errorText}
+                        value={this.props.value}
```

Comment:
Should we be passing `ref={this.props.innerRef}` here?

Reply:
Passing `ref={this.props.innerRef}` here will give us the `ref` of `BasePicker` component instance, not the DOM element we should get to obtain `focus()`.

Reply:
Hmm one issue that I found is that submitting the form without a value does not show the error message. It seems that  `AddressSearch` has the same problem and I'll address that in a different issue.

https://user-images.githubusercontent.com/22219519/156443068-b8e599ad-c3de-49fa-ade1-0350428c15f4.mov

Reply:
Ah ok that makes sense!

Reply:
Once [pull request](https://github.com/lawnstarter/react-native-picker-select/pull/464/files) on `react-native-picker-select` is merged and we update it the issue about the `ref` will be finally done. I was thinking about [npm overrides](https://github.com/npm/rfcs/blob/main/accepted/0036-overrides.md) to force `react-native-picker-select` to use the newer version, but it only works on `npm` version `8.3.0` and I am not sure if that really works.

Reply:
Yea, let's not do any overrides. Hopefully that `react-native-picker-select` PR will be merged soon. Otherwise we might have to figure out another solution.